cmd/packagesfuncs.go

```diff
@@ -222,7 +222,7 @@ func getModules() (standard, nonstandard, unknown []moduleInfo, err error) {
 			nonstandard = append(nonstandard, caddyModGoMod)
 		}
 	}
-	return
+	return standard, nonstandard, unknown, nil
```

Comment:
I'm not sure here what's the expected behaviour, but this change deviates from the existing code which does:
```suggestion
	return standard, nonstandard, unknown, err
```
Which means that if the last `caddy.Modules` we've ranged over above has returned an error when doing `GetModule`, then this will return a non-nil error with the current code, whereas with the proposed change it would still return `nil`.

Overall this seems like a deeper bug lurking...